v2/cf-nodejs-logging-support/src/lib/config/config-core.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
Except `logger` these fields are CF specific. You can move the to the `config-cf.json` file. Btw. to get a better impression on how these fields are supposed to work, you can check out the [field list ](https://help.sap.com/viewer/ee8e8a203e024bbb8c8c2d03fce527dc/Cloud/en-US/48b726c3f7534285b05eb31b5b7dc14d.html)in Application Logging docs.